src/components/Navbar.module.css

```diff
@@ -1,32 +1,136 @@
 @import url("https://fonts.googleapis.com/css2?family=Inter:wght@300;600&family=Maven+Pro:wght@500&display=swap");
+.coName {
+  flex: 2;
+  font-size: 30px;
+}
 
-.logo {
-  margin-top: 2px;
-  margin-left: -20px;
+.logoContainer {
+  display: flex;
+  flex: 0.75;
+  justify-content: center;
+  align-items: center;
 }
 
 .logoAndName {
   display: flex;
-  align-items: center;
-  gap: 50px;
-  font-size: 35px;
+  flex: 3.2;
 }
 
 .navbar {
   display: flex;
   align-items: center;
-  justify-content: space-around;
-  gap: 100px;
   background-color: rgb(0, 62, 82);
-
   color: white;
-  height: 70px;
+  height: 3em;
   font: bolder 25px "Maven Pro", sans-serif;
 }
 
 .navLinks {
   display: flex;
-  gap: 50px;
+  flex: 7;
+  justify-content: space-evenly;
+  align-items: center;
+  flex-wrap: wrap;
+  gap: 10px;
+}
+
+@media screen and (max-width: 1008px) {
+  .navLinks {
+    display: none;
+  }
+
+  .hamburgerMenuContainer {
+    border: white 1px solid;
```

Comment:
the border shorthand goes border: width style color so this should be 1px solid white